aspnetcore/mvc/views/view-components.md

```diff
@@ -104,13 +104,13 @@ Pascal-cased class and method parameters for Tag Helpers are translated into the
 </vc:[view-component-name]>
 ```
 
-Note: In order to use a View Component as a Tag Helper, you must register the assembly containing the View Component using the `@addTagHelper` directive. For example, if your View Component is in an assembly called "MyWebApp", add the following directive to the `_ViewImports.cshtml` file:
+Note: In order to use a View Component as a Tag Helper, you must register the assembly containing the View Component using the `@addTagHelper` directive. For example, if your view component is in an assembly called `MyWebApp`, add the following directive to the `_ViewImports.cshtml` file:
```

Comment:
- ~~In order~~
- ~~you must~~
- Let's be consistent all throughout with the casing of "view component".
- Italicize the file name.
- While you're in the file, make the **Invoking a view component as a Tag Helper** section display ONLY for 1.1+.

Reply:
> Let's be consistent all throughout with the casing of "view component".

That's what I was trying to do. It was lowercase almost everywhere.

I'll get these on the next commit. :+1:

Reply:
I want to simplify the prep phrase + subordinate conjunction just like we did the other day on that other PR. Let's just go with "If ... blah blah blah" ... "for example" is overused and obvious.